src/deterministic_brb.rs

```diff
@@ -245,6 +245,33 @@ impl<A: Actor<S>, SA: SigningActor<A, S>, S: Sig, BRBDT: BRBDataType<A>>
         Ok(packets)
     }
 
+    /// Resend any RequestValidation packets that have not yet received enough signatures.
+    #[allow(clippy::type_complexity)]
+    pub fn resend_pending_validation_requests(
+        &self,
+    ) -> Result<Vec<Packet<A, S, BRBDT::Op>>, Error<A, S, BRBDT::ValidationError>> {
+        let mut packets = Vec::new();
+        for (msg, sigs) in self.pending_proof.iter() {
```

Comment:
pending_proof is not inserted into until SignedValidated is received.  So I believe this still leaves a window between the broadcast in exec_op() until we get SignedValidated.

I am thinking that we need to somehow mark our actor/self with "source-op-in-flight" inside exec_op, and remove that after supermajority-of-supermajority.

could be I'm missing something though... ;-)

Reply:
> pending_proof is not inserted into until SignedValidated is received. So I believe this still leaves a window between the broadcast in exec_op() until we get SignedValidated.

With the short-circuiting in exec_op, we will process the RequestValidation and SignedValidated from ourselves before we return.

Reply:
I see.   so pending_proof is inserted into before exec_op() returns.  ok, that works afaict.  nice.

yeah, that'd be a good thing to comment in exec_op also, cuz not so obvious.